CMakeLists.txt

```diff
@@ -41,10 +41,12 @@ set(EMCC_ARGS
   -O3
   -s ALLOW_MEMORY_GROWTH=${ALLOW_MEMORY_GROWTH}
   -s EXPORTED_RUNTIME_METHODS=["UTF8ToString"]
+  -s EXTRA_EXPORTED_RUNTIME_METHODS=["addFunction"]
```

Comment:
is this needed? where is addFunction called from?

Reply:
If I understand correctly, this is needed to create a function pointer... Here's an example with `btDyamicsWorld.setInternalTickCallback`

```js
function callback(world, timestep) {}

const pointer = Ammo.addFunction(callback, Ammo.INTERNAL_TICK_CALLBACK_SIGNATURE)

world.setInternalTickCallback(pointer)
```

Reply:
Ah yes, makes sense.